app/tests/test_download.py

```diff
@@ -38,17 +38,10 @@ def test_invalid_id_raises_404(self, client: FlaskClient):
         assert response.status_code == 404
 
     @mock_aws
-    def test_download_record_standard_user_with_citable_reference_with_file_extension(
```

Comment:
we still need the test cases for `citable_reference_with_file_extension` and `citable_reference_without_file_extension` unless we move that logic into a helper function and test that